### PR TITLE
pi_task_callback, pi_task_wait_on and pi_task_push  are now inlined

### PR DESCRIPTION
**Pull request by haugoug**
Could we switch these functions to static inline ? This allows me to call them from C interrupt handler on pulpos, as they are simple.

### CODE REVIEW

**Review by gwtsivasiva — APPROVED** [review]
FIne by me.

**Review by antoine-faravelon — APPROVED** [review]
fine by me too.

**Comment by haugoug** [comment]
Ok you will need to add the corresponding inline functions in pmsis drivers, I let you merge this branch once this is done or I merge it now ?

**Comment by gwtsivasiva** [comment]
You can merge it now.

**Pull request merged**
